### PR TITLE
Librato MD Support

### PR DESCRIPTION
**Pull request by bryanmikaelian**

### CODE REVIEW

**Comment by bryanmikaelian** [comment]
@mheffner This should be good to go. We are now sending the payload to `/v1/measurements` if the config value has tags defined, otherwise we will send to `/v1/metrics`. 

Do you think we should keep this out of master till MD is GA?

File: lib/librato.js [hunk 1]
```diff
@@ -174,6 +177,44 @@ var post_metrics = function(ts, gauges, counters)
...
 
+var post_measurements = function(ts, measurements) 
+{
+  var payload = {
+    time: ts,
+    tags: tags,
+    measurements: measurements
+  };
+  
+  var parsed_host = url_parse(api || 'https://metrics-api.librato.com');
```
Comment (mheffner) [inline lib/librato.js]: Can we DRY this out with the other definition of this?

File: lib/librato.js [hunk 2]
```diff
@@ -267,23 +314,35 @@ var flush_stats = function librato_flush(ts, metrics)
...
-
-    if (mType == 'counter') {
-      counters.push(measure);
+    
+    if (multidimensional) {
+      measurements.push(measure);
+      
+      // Post MD measurement
+      if (measurements.length >= maxBatchSize) {
+        post_measurements(measureTime, measurements);
```
Comment (mheffner) [inline lib/librato.js]: I don't see a matching call to this at the end of flush() for any remaining measurements left in the payload.

Comment (bryanmikaelian) [inline lib/librato.js]: Oops. Probably a good idea to include this. Good catch!

**Comment by bryanmikaelian** [comment]
@mheffner All fixed up. I feel like `post_metrics` signature is a little awkward but it is probably fine.

File: README.md [hunk 3]
```diff
@@ -237,6 +237,25 @@ That configuration will proxy requests via a proxy listening on
...
+  "librato" : {
+    "tags": { "os" : "ubuntu", "hostname" : "production-web-server-1", ... }
+  }
+}
+```
+
+Once your config has been updated, all metrics submitted to Librato will inclue your defined tags.
+If a source name exists, we will automatically submit the source as a tag as well. 
+
+By default, this functionality is disabled and is newly supported by Librato. If you are interested in using this feature, you can reach out to our support team and request access.
```
Comment (mheffner) [inline README.md]: add `support@librato.com` email?

**Comment by mheffner** [comment]
LGTM

**Comment by bryanmikaelian** [comment]
Closing this PR in favor ofhttps://github.com/librato/statsd-librato-backend/pull/59 since we can now add per-stat tagging

**Pull request closed without merging**